js/widgets/phrasemaker.js

```diff
@@ -3971,19 +3950,19 @@ function PitchTimeMatrix() {
                 cellTuplet.setAttribute("id", j);
             }
 
-            __mouseDownHandler = function(event) {
-                that._mouseDownCell = event.target;
+            const __mouseDownHandler = (event) => {
+                this._mouseDownCell = event.target;
             };
 
-            __mouseUpHandler = function(event) {
-                that._mouseUpCell = event.target;
-                if (that._mouseDownCell !== that._mouseUpCell) {
-                    that._tieNotes(that._mouseDownCell, that._mouseUpCell);
+            const __mouseUpHandler = (event) => {
+                this._mouseUpCell = event.target;
+                if (this._mouseDownCell !== this._mouseUpCell) {
+                    this._tieNotes(this._mouseDownCell, this._mouseUpCell);
                 } else {
-                    let nodes = Array.prototype.slice.call(
+                    const nodes = Array.prototype.slice.call(
                         this.parentElement.children
                     );
-                    that._createpiesubmenu(
+                    this._createpiesubmenu(
                         nodes.indexOf(this),
                         this.getAttribute("alt"),
```

Comment:
`parentElement`, `getAttribute` are properties of the caller, not `this` class' object. Kind of a chain reaction from here. Please reevaluate your arrow function conversions. Click on the time signatures at the bottom of the widget to reproduce the crash here.